docs/_data/table/cluster-management.csv

```diff
@@ -1,16 +1,16 @@
 propertyName,defaultValue
 alluxio.integration.master.resource.cpu,1
 alluxio.integration.master.resource.mem,1024 MB
-alluxio.integration.mesos.executor.dependency.path,https://s3.amazonaws.com/alluxio-mesos
-alluxio.integration.mesos.jre.path,jre1.7.0_79
-alluxio.integration.mesos.jre.url,https://s3.amazonaws.com/alluxio-mesos/jre-7u79-linux-x64.tar.gz
+alluxio.integration.mesos.executor.dependency.path,http://downloads.alluxio.org/downloads/files/${alluxio.version}/alluxio-${alluxio.version}-bin.tar.gz
+alluxio.integration.mesos.jdk.path,jdk1.7.0_79
+alluxio.integration.mesos.jdk.url,https://s3-us-west-2.amazonaws.com/alluxio-mesos/jdk-7u79-linux-x64.tar.gz
 alluxio.integration.mesos.master.name,AlluxioMaster
 alluxio.integration.mesos.master.node.count,1
-alluxio.integration.mesos.principal,
+alluxio.integration.mesos.principal,alluxio
 alluxio.integration.mesos.role,*
 alluxio.integration.mesos.secret,
-alluxio.integration.mesos.user,root
+alluxio.integration.mesos.user,
```

Comment:
setting it as `root` will run the Alluxio Master/Worker as `root`, while setting it to `""` will cause Mesos to determine what's the appropriate account to run as.